pkg/network/ebpf/c/protocols/kafka/types.h

```diff
@@ -1,17 +1,42 @@
 #ifndef __KAFKA_TYPES_H
 #define __KAFKA_TYPES_H
 
+#include "defs.h"
+
 typedef enum {
     KAFKA_PRODUCE = 0,
     KAFKA_FETCH
 } __attribute__ ((packed)) kafka_operation_t;
 
 typedef struct {
-    s32 message_size;
```

Comment:
The command `CC=clang go tool cgo -godefs -- -I ../../ebpf/c -I ../../../ebpf/c -fsigned-char kafka_types.go | go run /git/datadog-agent/pkg/ebpf/cgo/genpost.go > kafka_types_linux.go` doesn't work with s32/s16 types